src/buildActionMaps.ts

```diff
@@ -81,7 +81,7 @@ export function buildActionMaps<Definitions extends EndpointDefinitions, Interna
   function buildQueryAction(endpoint: string, definition: QueryDefinition<any, any, any, any>) {
     const queryAction: StartQueryActionCreator<any> = (
       arg,
-      { subscribe = true, forceRefetch = false, refetchOnMountOrArgChange = false, subscriptionOptions } = {}
+      { subscribe = true, forceRefetch = false, refetchOnMountOrArgChange, subscriptionOptions } = {}
```

Comment:
Hm. Did I accidentally change the meaning of this in the non-hooks-case or is that okay? :thinking: